packages/lit-dev-content/site/docs/v3/components/properties.md

```diff
@@ -553,11 +566,16 @@ If this behavior doesn't fit your use case, there are a couple of options:
 
 You can configure a property so that whenever it changes, its value is reflected to its [corresponding attribute](#observed-attributes). Reflected attributes are useful because attributes are visible to CSS, and to DOM APIs like `querySelector`.
 
+Setting `useDefault` to true prevents the property's default value from initially reflecting to the corresponding attribute. All subsequent changes will be reflected. This matches web platform behavior for attributes like `id`. For example, the default value of an element's `id` property is `''` (an empty string) and initially it does not have an `id` attribute, but if the `id` property is set (even to an empty string), the appropirate `id` attribute is reflected. In addition, if the attribute is removed, the property will be reset to its default value.
```

Comment:
Need to add a bit about how removing an attribute restores the property to the default. It'd be good to sync the language here and the jsdocs for the option a bit.

Reply:
I think the preview may be broken because the property options in the example use `useDefualt` and that's not published yet.

Reply:
Replaced cast to fix prview